src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -597,3 +601,18 @@ fn update_component_log_level(config: &TomlConfig) {
         updated_inner
     });
 }
+
+fn determine_jailed(config: &TomlConfig) -> Result<String> {
+    let Some(hypervisor_config) = config.hypervisor.get(&config.runtime.hypervisor_name) else {
```

Comment:
Does this compile? It kind of looks like `if let` was meant but the following code seems to contradict that.

Reply:
Oh I missed that addition to the language, thanks! :-)